src/ZendDiagnostics/Check/MySQLCheck.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * @license   http://framework.zend.com/license/new-bsd New BSD License
+ */
+
+namespace ZendDiagnostics\Check;
+
+use PDO;
+use ZendDiagnostics\Result;
+
+/**
+ * Ensures a connection to the MySQL server/database is possible.
+ */
+class MySQLCheck implements CheckInterface
```

Comment:
@lsmith77 good q, I can change this and remove MySQL specific bits. Let me know!